test/plots/rectanglePlotTests.ts

```diff
@@ -142,6 +142,17 @@ describe("Plots", () => {
         assert.strictEqual(entities[1].index, 2, "the entity of index 2 is retrieved");
         svg.remove();
       });
+
+      it("retrieves undefined from entityNearest() when no entities are rendered", () => {
```

Comment:
Can we use a consistent name for the tests? `"retrieves undefined from entityNearest when no entities are rendered"` seems best.